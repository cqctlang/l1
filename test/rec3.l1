@record r {a, b};
tr = r();
printf("%a\n", tr.a);
printf("%a\n", tr.b);
