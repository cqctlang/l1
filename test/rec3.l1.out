entry: var params, 2 locals, 0 temps
a: var params, 1 locals, 0 temps
b: var params, 1 locals, 0 temps
<nil>
<nil>
